src/gui/protocolwidget.ui

```diff
@@ -53,6 +40,9 @@
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
+     <property name="accessibleName">
```

Comment:
Hmm the tab is added in code but the description in the .ui?

Reply:
See next line:
```
<string>Local activity table</string>
```
It's the name for the table widget.